PLAN.md

```
@@ -0,0 +1,250 @@
+- [ ] OpenRouter model statistics
+- [ ] LiveCodeBench integration (coding benchmarks)
+- [ ] Arena-Hard integration (chat benchmarks)
+- [ ] GitHub Actions workflow for daily data refresh
+
+**Impact:** Model rankings stay current automatically
+
+### 2.2 Scoring Algorithm
+**Goal:** Implement weighted scoring from VIRTUAL_MODELS_PLAN.md
+
+Formulas to implement:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)